snooty/parser.py

```diff
@@ -1597,12 +1597,19 @@ def propagate_facets(self) -> None:
         for base, _, files in os.walk(root):
             if "facets.toml" in files:
                 facet_path = Path(os.path.join(base, "facets.toml"))
-                curr_facet, diagnostics = self.config.load_facet_file(facet_path)
+                curr_facet, diagnostics = self.config.load_facets_from_file(facet_path)
+
+                if curr_facet is None:
+                    logger.warning(
```

Comment:
I think this actually sounds like a `logger.error()`

But would it be a diagnostic instead of just a logging message?

Reply:
Yes, that's correct! It would be a diagnostic, and it should already be included in the result. I'll remove this log.